aleph/views/serializers.py

```diff
@@ -312,3 +313,60 @@ def _serialize(self, obj):
 
 class MappingSerializer(Serializer):
     pass
+
+
+class VisDiagramEntitySerializer(EntitySerializer):
```

Comment:
Let's not use the term Vis in the backend. DiagramEntity.. is fine :)

Reply:
done